pages/common/git-worktree.md

```diff
@@ -0,0 +1,15 @@
+# git worktree
+
+> Manage multiple working trees attached to the same repository.
+
+- Create a new folder with the specified branch checked out into it:
+
+`git worktree add {{path/to/folder}} {{branch}}`
```

Comment:
Please fix the inconsistency of options order in your 1st and 3rd examples. 

First one has `add {{folder}} {{branch}}` 
Third has `add -b {{branch}} {{folder}}`

Also, please reorder the examples to have the `add` commands first and then the `list` command.